browse/static/js/catalyzex.js

```diff
@@ -0,0 +1,64 @@
+  }
+
+  const fetchCatalyzeXCode = async () => {
+    const cxApiUrl = `https://www.catalyzex.com/api/code?src=arxiv&paper_arxiv_id=${arxivId}`;
+
+    let result = {};
+
+    try {
+      result = await $.ajax({ url: cxApiUrl, timeout: 2000, dataType: "json" });
+    } catch (error) {
+      result = {};
+    }
+
+    return result;
+  };
+
+  $output.html('');
+
+  const { count: implementations, cx_url: cxImplementationsUrl } = await fetchCatalyzeXCode()
+
+  $output.append("<h2>CatalyzeX</h2>");
+
+  const addCodeURL = new URL("https://www.catalyzex.com/add_code");
+  addCodeURL.searchParams.set('title', paperTitle);
+  addCodeURL.searchParams.set('paper_url', paperUrl);
+
+  const submitItHereLink = `<a target="_blank" href="${addCodeURL}">submit it here</a>`;
+
+  if (implementations) {
+    const codeLink = $(`<a target="_blank"></a>`);
+    codeLink.attr("href", cxImplementationsUrl);
+    codeLink
+      .append(icons.github)
+      .append(`${implementations} code implementation${implementations > 1 ? "s" : ""} found on`)
+      .append(icons.catalyzex)
+      .append("CatalyzeX");
+
+    $output
+      .append(codeLink)
+      .append($(`<p>If you have code to share with the community, please ${submitItHereLink}.</p>`))
+  } else {
+    $output.append(`<p>No code found just yet. If you have code to share with the community, please ${submitItHereLink}.</p>`)
```

Comment:
is there a way to make the "submit it here" link bold only just in this case? like so
![image](https://user-images.githubusercontent.com/978669/216736016-897a438b-7c3f-470b-9b0b-90edfa68017f.png)

Reply:
For sure! I've just updated it